places-launch-rules/define-data-elements.md

```diff
@@ -0,0 +1,65 @@
+# Defining Data Elements
+
+### Data Elements
+
+Data elements are the building blocks for the applications data dictionary and are used to collect, organize, and deliver data across marketing and ad technology.
+
+A data element is a variable where the value can be mapped to a Visitor ID, a Carrier Name, an Advertising ID, a Push ID and so on. In Experience Platform Launch, you can reference this value by its variable name. This collection of data elements becomes the dictionary of defined data that you can use to build your rules \(events, conditions, and actions\), and this dictionary is shared across Experience Platform Launch where it can be used with any extension in your property.
+
+With Places extension, you can reference values from the following targets:
+
+* Current POI, which refers to the POI in which your customer is currently located. If the user is located in multiple POIs, the one belongs to the library with higher rank is picked. If multiple POIs belong to the highest rank library, the one with the smallest radius is picked.
```

Comment:
Please move the content starting with the 2nd sentence down one line and line it up with the bullet. It should look like this:
"* Current POI, which refers to the POI in which your customer is currently located. 

    If the user is located in multiple POIs, the one belongs to the library with higher rank is picked. If multiple POIs belong to the highest rank library, the one with the smallest radius is picked.